meta-balena-dunfell/recipes-core/systemd/systemd/0003-Don-t-run-specific-services-in-container.patch

```diff
@@ -53,4 +53,4 @@ index 38a7f26..43f171e 100644
 +
  [Service]
  BusName=org.freedesktop.login1
- CapabilityBoundingSet=CAP_SYS_ADMIN CAP_MAC_ADMIN CAP_AUDIT_CONTROL CAP_CHOWN CAP_KILL CAP_DAC_READ_SEARCH CAP_DAC_OVERRIDE CAP_FOWNER CAP_SYS_TTY_CONFIG
```

Comment:
The purpose of modifying this patch to change these capabilities is unclear to me. Would it make sense to split this into a separate commit and further explain the change?

Reply:
@jakogut This diff is misleading, all I did was update the patches to match the new upstream sources. So the `CapabilityBoundingSet` field actually changed upstream, and our patch has to reflect that change in order to inject `ConditionVirtualization=!docker` without fuzzy matching.

The patch itself does not change `CapabilityBoundingSet` if you view the patch file as raw.